docs/application/dotnet/guides/nui/flexcontainer.md

```diff
@@ -412,20 +438,23 @@ You can modify the flex item placement through the following properties:
 
     To add space around an item:
 
-    ```
+    ```cs
     /// Create the flex container
     FlexContainer flexContainer = new FlexContainer();
+    Window.Instance.GetDefaultLayer().Add(flexContainer);
 
     /// Create a flex item
     View item = new View();
 
     /// Add a margin around the item
     item.FlexMargin = new Vector4(10.0f, 10.0f, 10.0f, 10.0f);
-
-    /// Add the item to the container
+    item.Size2D = new Size2D(200, 200);
+    item.BackgroundColor = Color.Red;
     flexContainer.Add(item);
     ```
 
 ## Related Information
```

Comment:
Global: Kindly ensure to follow sentence case for all headings. 
@mijin85cho what's your opinion? 

```suggestion 
## Related information

Reply:
@mijin85cho thanks a lot!